src/Tests/RegressionTests/RegressionTests.fs

```diff
@@ -11,14 +12,20 @@ let actualsPath = "../../../Actual"
 module Check =
 
     let fromFilename(output : ITestOutputHelper) (filename : string) =
-        let expected = new XLWorkbook(Path.Combine(expectedsPath, filename))
         let actual = new XLWorkbook(Path.Combine(actualsPath, filename))
+        let expected = new XLWorkbook(Path.Combine(expectedsPath, filename))
         Assert.Workbook.Equal(expected, actual, filename, output)    
 
 type Tests(output : ITestOutputHelper) =
+    do
+        if RuntimeInformation.IsOSPlatform(OSPlatform.Linux) then
+            LoadOptions.DefaultGraphicEngine <- ClosedXML.Graphics.DefaultGraphicEngine("DejaVu Sans")
```

Comment:
This works around the fact that Calibri font is not available on Linux build agents